### PR TITLE
A single program to perform 8 arithmetic operations

### PR DESCRIPTION
**Pull request by akhil-kk**
The program takes 2 numbers as input and perform  multiply / divide / modulus / substract / greater than / less than / is equal/ operations

### CODE REVIEW

File: 002arithmetic_operations.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+function add(number1, number2) {
```
Comment (kuism) [inline 002arithmetic_operations.js]: Convert all the functions to es6

Comment (kuism) [reply 002arithmetic_operations.js]: `add = (number1, number2) => {

} `
Like this

File: 002arithmetic_operations.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    let difference = number1 - number2;
+    console.log("The difference is ", difference);
+}
+
+const multiply = (number1, number2) => {
+    let product = number1 * number2;
+    console.log("The product is ", product);
+}
+
+const devide = (number1, number2) => {
```
Comment (kuism) [inline 002arithmetic_operations.js]: spelling mistake

Comment (akhil-kk) [reply 002arithmetic_operations.js]: shall i rename devide ==> division

**Pull request merged**